main/src/cgeo/geocaching/connector/gc/GCLoggingManager.java

```diff
@@ -14,25 +24,15 @@
 import cgeo.geocaching.enumerations.LogType;
 import cgeo.geocaching.enumerations.StatusCode;
 import cgeo.geocaching.loaders.UrlLoader;
+import cgeo.geocaching.models.Geocache;
+import cgeo.geocaching.models.Image;
+import cgeo.geocaching.models.TrackableLog;
```

Comment:
Sorry, it's ok. I am used to see the current package located at the top and was somehow confused by the diff.